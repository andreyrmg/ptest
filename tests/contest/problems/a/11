10
25 D
32 B
10 1
24 21
35 1E
2 11000
28 19
6 112
19 29
17 4
