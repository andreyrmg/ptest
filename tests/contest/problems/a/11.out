10 1
17 4
32 B
25 D
2 11000
28 19
6 112
19 29
24 21
35 1E
